worlds/wargroove2/__init__.py

```diff
@@ -0,0 +1,194 @@
+        self.level_list = low_victory_checks_levels[0:4] + non_starting_levels
+
+        # Final Levels
+        self.final_levels = get_final_levels(self.player)
+        final_levels_no_ocean = list(level for level in self.final_levels if not level.has_ocean)
+        final_levels_ocean = list(level for level in self.final_levels if level.has_ocean)
+        random.shuffle(final_levels_no_ocean)
+        random.shuffle(final_levels_ocean)
+        non_north_levels = final_levels_ocean + final_levels_no_ocean[1:]
+        random.shuffle(non_north_levels)
+        self.final_levels = final_levels_no_ocean[0:1] + non_north_levels
+
+        # Selecting a random starting faction
+        if self.options.commander_choice == "random_starting_faction":
+            factions = [faction for faction in faction_table.keys() if faction != "Starter"]
+            starting_faction = Wargroove2Item(self.multiworld.random.choice(factions) + ' Commanders', self.player)
+            self.multiworld.push_precollected(starting_faction)
+
+    def create_items(self) -> None:
+        # Fill out our pool with our items from the item table
+        pool = []
+        precollected_item_names = {item.name for item in self.multiworld.precollected_items[self.player]}
+        ignore_faction_items = self.options.commander_choice == "locked_random"
+        for name, data in item_table.items():
+            if data.code is not None and name not in precollected_item_names and \
+                    not data.classification == ItemClassification.filler:
+                if name.endswith(' Commanders') and ignore_faction_items:
+                    continue
+                item = Wargroove2Item(name, self.player)
+                pool.append(item)
+
+        for i in range(0, 5):
+            pool.append(Wargroove2Item("Commander Defense Boost", self.player))
+            pool.append(Wargroove2Item("Income Boost", self.player))
+
+        # Matching number of unfilled locations with filler items
+        total_locations = len(self.first_level.location_rules.keys())
+        for level in self.level_list[0:LEVEL_COUNT]:
+            total_locations += len(level.location_rules.keys())
+        locations_remaining = total_locations - len(pool)
+        while locations_remaining > 0:
+            # Filling the pool equally with the groove boost
+            pool.append(Wargroove2Item("Groove Boost", self.player))
+            locations_remaining -= 1
+
+        self.multiworld.itempool += pool
+
+        victory = Wargroove2Item("Wargroove 2 Victory", self.player)
+        for i in range(0, 4):
+            final_level = self.final_levels[i]
+            self.get_location(final_level.victory_locations[0]).place_locked_item(victory)
+        # Placing victory event at final location
+        self.multiworld.completion_condition[self.player] = lambda state: \
+            state.has("Wargroove 2 Victory", self.player, self.options.final_levels)
```

Comment:
Just so this is explicitly an int
```suggestion
            state.has("Wargroove 2 Victory", self.player, self.options.final_levels.value)
```